draft-roadmap-2024.md

```diff
@@ -1,49 +1,81 @@
 The roadmap is one of the important artifacts on the website.
 
-The draft roadmap 2024 was created based on information from the Attention Ubiquity Core Team https://github.com/ubiquity
+The draft roadmap was created based on information from the Attention Ubiquity Core Team https://github.com/ubiquity and Alex's recommendations https://github.com/ubiquity/research/pull/130#discussion_r1485777642
+
 Once the content and structure of the roadmap are approved, a visually appealing roadmap will be created.
 
-The draft of the roadmap 2024 content and structure for discussion:
+The draft of the roadmap Q1 2024 - Q2 2025 for discussion:
 
 ## Product Development
 
-### Q1
+### Q1 2024
+
+Security assessment
+
+### Q2 2024
+
+Ubiquity Dollar stable version launch
+
+Ubiquity Card integration to DAO OS
+
+### Q3 2024
+
+DAO OS Kernel stable version launch
+
+Ubiquity Dollar integration to DAO OS
+
+### Q4 2024
+
+DAO OS Plugins open for contribution
+
+DAO OS AI agents alpha version 
+
+### Q1 2025
+
+Ubiquity Bank alpha version 
```

Comment:
The roadmap usually indicates when an item is available to users.

If we are serious about becoming a Web3 bank for individuals, then it makes sense to provide standard banking services for them https://en.wikipedia.org/wiki/Bank

If by “metaverse bank” we mean something non-usual, then we need to determine what exactly it is.

I think that the alpha version of the bank should have at least secure user accounts, payment cards, and vaults.
Other services such as cashback, loans, currency exchange, and others may be provided later.

Reply:
Most of what you mentioned comes out of the box with DeFi. We don't really need to build anything except for some UI integrations and then the whole payment card bit.

Reply:
@pavlovcik As far as I understand, the vision you talked about is something like “earn in Metaverse, spend conveniently everywhere”:
1) Earn 
- By completing tasks
- By providing capital
- Analyze everything in your account
2) Spend
- Using a card

If so, then some functionality is missing. This could be a solution like
- Proxy Yield Aggregator you mentioned a while ago (maybe like Beefy)
- automation of LP on Uniswap which we once discussed (like Gamma)
- specialized Vaults that I offered (like PLAY)
- integration with certain DeFi protocols (like Dai Savings Rate (DSR) of MakerDao)
- direct use of DeFi pools (Ubiquity just provides a dashboard)
- or something else